tests/performance/table_function_null.xml

```diff
@@ -0,0 +1,3 @@
+<test>
+    <query>INSERT INTO function null('number UInt64') SELECT * FROM numbers_mt(1000000000);</query>
```

Comment:
Why not to make functional test instead of performance test?

Reply:
I add a performance test because, in PR #14120, alexey-milovidov have a comment. I can also add a functional test if we need.

```
TODO: Add performance test at least for Null.
Something like INSERT INTO null() SELECT * FROM numbers_mt(1000000000).
```

Reply:
But it duplicates [performance test](https://github.com/ClickHouse/ClickHouse/blob/9b6c62e82b2b59cb7b63869970a2af4a2d4143ef/tests/performance/insert_parallel.xml#L6) from the PR you mentioned. Maybe replace `INSERT INTO t ...` with `INSERT INTO FUNCTION null('number UInt64') ...` in `insert_parallel.xml` or rewrite `table_function_null.xml` to functional test?

Reply:
Sorry I didn't realize this problem. I have rewrite `table_function_null.xml` to a functional test.